src/main/resources/static/test-scheme.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE Article (
+  id INTEGER NOT NULL IDENTITY,
+  title VARCHAR(45) NOT NULL,
+  label VARCHAR(45) NOT NULL,
+  text VARCHAR(200) NOT NULL,
+  created_date DATE NOT NULL,
```

Comment:
I want that this date will be automatic fill on DB level when record creates and add modified_date, that have to updates each time when some record changes